src/Symfony/Component/Config/Definition/Builder/ArrayNodeDefinition.php

```diff
@@ -371,7 +376,7 @@ protected function createNode(): NodeInterface
 
             if ($this->default) {
                 if (!\is_array($this->defaultValue)) {
-                    throw new \InvalidArgumentException(\sprintf('%s: the default value of an array node has to be an array.', $node->getPath()));
+                    throw new \InvalidArgumentException(\sprintf('"%s": the default value of an array node has to be an array.', $node->getPath()));
```

Comment:
this should be reverted. It is a case where the fabbot check is wrong.

Reply:
ah ok, I didn't know

Reply:
@stof reverted